Makefile

```diff
@@ -91,3 +106,21 @@ get-kubectl: ## Download kubectl only if it's not available. Used in the docker
 .PHONY: install
 install: build
 	go install ./...
+
+# Generate mocks from the interfaces.
+.PHONY: mocks
+mocks:
+	@if [ ! -f $(GOPATH)/bin/mockery ]; then \
+		echo "installing mockery ..." && \
+ 		env GO111MODULE=off $(GO) get -u github.com/vektra/mockery/.../; \
```

Comment:
I'm not sure that I want a `Makefile` to install dependencies onto my system. 1/5 but our other dependencies, e.g. Helm, are just listed on the README so that the developer can install and manage them him or herself.

In my opinion, I think it's probably OK to just assume `mockery` is installed in the `Makefile`.